dpctl/tensor/_reshape.py

```diff
@@ -86,7 +86,7 @@ def reshape(X, newshape, order="C", copy=None):
         raise TypeError
     if not isinstance(newshape, (list, tuple)):
         newshape = (newshape,)
-    if order not in ["C", "F"]:
+    if order not in "cCfF":
```

Comment:
This is not sufficient. The keyword argument must also be normalized to the upper case.

The code below makes the assumption that `order` is either "C" or "F".